src/onegov/town6/locale/de_CH/LC_MESSAGES/onegov.town6.po

```diff
@@ -635,6 +633,9 @@ msgid "Something missing? Propose a new entry."
 msgstr ""
 "Fehlt ein Eintrag im Verzeichnis? Schlagen Sie einen neuen Eintrag vor."
 
+msgid "Internal Notes"
```

Comment:
I also found these missing translations used in 'chats' and mentioned it to @BreathingFlesh.